src/util.js

```diff
@@ -84,8 +84,8 @@ export function readJSONFile(filePath, fallbackValue) {
  */
 export function readFileFromCache(filePath, fallbackValue) {
     debug(`read file from cache: ${filePath}`);
-    return (path.extname(filePath) === '.json' ? readJSONFile : readFile)
-        .call(null, path.join(getCacheFolder(), filePath), fallbackValue);
+    return (path.extname(filePath) === '.json' ?
```

Comment:
Expected an identifier and instead saw 'return'.<br>Expected an operator and instead saw '('.<br>Expected ')' and instead saw 'path'.